docs/data/charts/funnel/funnel.md

```diff
@@ -29,6 +29,17 @@ To disable the legend, set the `hideLegend` prop to `true`.
 
 {{"demo": "FunnelLegend.js"}}
 
+## Pyramid Chart
+
+The pyramid chart is a variation of the funnel chart.
+
+To create a pyramid chart, set the `curve` property to `pyramid` in the series.
+It will automatically adjust the sections to create a pyramid shape.
+
+The pyramid chart is useful for displaying data that has a natural hierarchy or progression.
```

Comment:
Same for the funnel 🤭

I'm biased on this one because for me pyramid should not exist. It removes all the data visualisation. On a pyramid, values are only visible in the label. The shape has no meaning.

Maybe best is to remove this sentence and let users pick the pyramid if their designer ask them one